### DISCUSSION TITLE
Allow Multi-line log-entries in Grafana Logs View to be collapsable and expandable

### DISCUSSION THREAD

**Discussion by ggruber4711** [grafana/grafana#46507] [Dashboards, Panels & Visualizations]
Promtail with the multiline stage is really useful to ship logs to Loki, even if the log-file contains exceptions (such as a typical Java stacktrace).

However one caveat from a UI perspective is still, that the huge log-entry containing the exception-stacktrace cannot be collapsed or expanded. Clicking on the expand button will just display the meta-data of the log-entry.

Neither of the grafana Log UI options, allows me to collapse the huge log-entry to just one line.

![image](https://user-images.githubusercontent.com/3330414/158055705-0a8468b7-b776-4da9-8351-72e7c0eca507.png)

**Comment by ethernoy** [comment]
This is a very useful feature and was once somehow possible in Grafana 7. I don't understand why wrap lines doesn't collapse/expand multi-line log in Grafana 9 ..

**Comment by himao** [comment]
Any progress or workaround?

**Comment by jcaceres** [comment]
Just mentioning that similar feature request was submitted in https://github.com/grafana/grafana/discussions/45802

**Comment by svennergr** [comment]
Please use https://github.com/grafana/grafana/issues/74261 for further discussion. Thank you!
